.github/workflows/test.yml

```diff
@@ -0,0 +1,31 @@
+## Test with multiple node versions.
+name: test
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+  pull_request:
+    branches:
+      - main
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [10, 12, 14, 16, 17]
```

Comment:
Also @trentm, what is your opinion on expanding this list?

We could add `18` and `19`. WDYT?

Reply:
Yes we could change that to `[10, 12, 14, 16, 18, 19]`.

Reply:
https://github.com/elastic/ecs-logging-nodejs/pull/132/commits/03752ef936c169f3a421e1603d5abd23a36a1713

Oh, no. It's also failing for 18 and 19.